pom.xml

```diff
@@ -69,6 +69,15 @@
             <version>RELEASE</version>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-web</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-web</artifactId>
+            <version>5.2.9.RELEASE</version>
+        </dependency>
```

Comment:
Adding needed dependency for the @RequestMapping annotation to work for your controller. This way, your index.html file can be shown when a user navigates to your root endpoint.